src/plugins/vis_type_timeseries/server/lib/vis_data/request_processors/series/date_histogram.test.js

```diff
@@ -86,8 +86,7 @@ describe('dateHistogram(req, panel, series)', () => {
             },
           },
           meta: {
-            bucketSize: 10,
```

Comment:
`bucketSize` was removed to avoid of any calculation in future, cause in fact we need a `interval`